lib/LedgerSMB/DBObject/Menu.pm

```diff
@@ -65,6 +66,22 @@ sub generate_section {
     return @{$self->{menu_items}};
 }
 
+=item generate_item()
```

Comment:
Ok. Are there any compelling scenarios to add this extra bit of code? Is this an optimization? If so, is it an optimization of response time, network data, developer time?

Given the amount of data to be sent over (218 items max), I'm thinking we'd be wise to optimize for developer time here and add API entry points only when we need them. My point: is this additional code currently being tested with unit tests? (If not, removing the code is quicker than implementing unit tests, I imagine?)